.github/workflows/test.yml

```diff
@@ -164,21 +166,22 @@ jobs:
             mkdir -p ${CI_DIST_IMAGES_OUTPUT}
             cp ${CI_IMAGE_CACHE}/${CI_IMAGE_PR_TAR}.gz ${CI_DIST_IMAGES_OUTPUT}/${CI_IMAGE_PR_TAR}.gz
             gunzip ${CI_DIST_IMAGES_OUTPUT}/${CI_IMAGE_PR_TAR}.gz
-            echo "::set-output name=PR_IMAGE_RESTORED::true"
+            echo "PR_IMAGE_RESTORED=true" >> "$GITHUB_OUTPUT"
         fi
 
     # only run the following steps if the PR image was not found in the cache
+    - name: Check out code into the Go module directory - from current pr branch
```

Comment:
same I see the ordering of steps has changed..

Reply:
Same here, setup-go wants to cache go modules. So we need to checkout before setting up, otherwise the go.sum is not available.